src/ontology/mondo-ingest.Makefile

```diff
@@ -321,7 +321,7 @@ documentation: j2 $(ALL_DOCS) unmapped-terms-docs mapped-deprecated-terms-docs s
 build-mondo-ingest:
 	$(MAKE) refresh-imports exclusions-all mondo-ingest.db slurp-all mappings matches \
 		mapped-deprecated-terms mapping-progress-report \
-		recreate-unmapped-components documentation
+		recreate-unmapped-components sync documentation
```

Comment:
## Makefile setup
@matentzn I'm not sure how you like this. But I set up like so:
- `build-mondo-ingest: ... sync`
- `sync: ... sync-subclassof`
- `sync-subclassof: reports/sync-subclassof.robot.template.tsv` (`sync-subclassof` is just an alias for the TSV)